documentation/asciidoc/accessories/display/troubleshooting.adoc

```diff
@@ -1,5 +1,44 @@
 == Troubleshooting the Display
 
+[discrete]
+=== Are you running Raspberry Pi OS bullseye and the display turns off during the boot sequence?
+
+There is a bug that requires a workaround to get the display working until the bug is fixed: https://github.com/raspberrypi/linux/issues/4686
+
+First, make sure the system is up-to-date and power-cycle the display to see if the update fixed the problem:
+
+[,bash]
```

Comment:
If it does get merged — which I'm still pondering — this bit will get shortened down considerably.